assets/common/utils/MicroEvent.js

```diff
@@ -13,7 +13,8 @@ const MicroEvent = () => {
      * @param {*} [payload] An optional event payload
      */
     fire(event, payload) {
-      if (debug) console.log(`Firing event: ${event}`, 'Payload: ', payload);
+        // eslint-disable-next-line no-console
```

Comment:
Why are these indented more than the line following them?

Reply:
_These_ as in this and the one a few lines below as well

Reply:
Indenting 2 spaces is hard with .editorconfig set to 4, okay?! :(